app/services/storage.js

```diff
@@ -534,17 +532,14 @@ class ProfileStorageSchemaV4 {
     const v = await storage.userVersion();
 
     if (v === this.version) {
-      console.log(`Storage already at version ${v}.`);
       return v;
     }
 
     if (v === 0) {
-      console.log(`Creating storage at version ${this.version}.`);
       return this.create(storage);
     }
 
     if (v < this.version) {
-      console.log(`Updating storage from ${v} to ${this.version}.`);
```

Comment:
I'd like to keep the creation and update branches unless you have strong feelings otherwise. Knowing when DB updates and creation happen is useful, and should only happen on first launch.